x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_results_table/use_exploration_results.ts

```diff
@@ -45,10 +44,14 @@ import { useExplorationDataGrid } from './use_exploration_data_grid';
 export const useExplorationResults = (
   dataView: DataView | undefined,
   jobConfig: DataFrameAnalyticsConfig | undefined,
-  searchQuery: estypes.QueryDslQueryContainer,
-  toastNotifications: CoreSetup['notifications']['toasts'],
-  mlApiServices: MlApiServices
+  searchQuery: estypes.QueryDslQueryContainer
 ): UseIndexDataReturnType => {
+  const {
+    services: {
+      notifications: { toasts },
+    },
+  } = useMlKibana();
+  const ml = useMlApiContext();
```

Comment:
Same here - not sure we need to use `useMlApiContext` since it should be included in `useMlKibana`